### PR TITLE
Deploy to Testnet

### PR DESCRIPTION
**Pull request by h0ngcha0**
Some updates while deploying to testnet

### CODE REVIEW

File: relayer/spy_relayer/.env.tilt [hunk 1]
```diff
@@ -14,11 +14,17 @@ RELAYER_CHAINS='[{
...
+  "chainName": "BSC",
+  "nativeCurrencySymbol": "BNB",
+  "nodeUrl": "http://bsc-devnet:8545",
+  "walletPrivateKeys": ["0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"]
 }, {
   "chainId": 255,
   "nativeCurrencySymbol": "ALPH",
   "nodeUrl": "http://alph-full-node:22973",
-  "walletPrivateKeys": ["vault alarm sad mass witness property virus style good flower rice alpha viable evidence run glare pretty scout evil judge enroll refuse another lava"]
+  "walletPrivateKeys": ["a642942e67258589cd2b1822c631506632db5a12aabcf413604e785300d762a5"]
```
Comment (polarker) [inline relayer/spy_relayer/.env.tilt:27]: Private keys are only for devnet?

Comment (h0ngcha0) [reply relayer/spy_relayer/.env.tilt:27]: Yeah, devnet only

**Comment by h0ngcha0** [comment]
> General question: if we want to deploy to mainnet, how much changes would it be?

The change should not be that big. We need to deploy all contracts to mainnet and update the corresponding configs. Infra should be almost the same between testnet and mainnet. Probably there will be some work to get our partners up and running for Guardians as well.

**Review by polarker — APPROVED** [review]
LGTM, ready fro merge

**Pull request merged**